app/src/components/species/components/SpeciesCard.tsx

```diff
@@ -1,45 +1,93 @@
-import { Chip, Stack, Typography } from '@mui/material';
+import { mdiCircle } from '@mdi/js';
+import Icon from '@mdi/react';
+import { Chip, colors, Stack, Typography } from '@mui/material';
+import { grey } from '@mui/material/colors';
+import React from 'react';
 
 interface ISpeciesCard {
-  commonName: string | null;
+  commonNames: string[];
   scientificName: string;
   tsn: number;
+  rank: string;
+  kingdom: string;
 }
 
 const SpeciesCard = (props: ISpeciesCard) => {
+  const rankColors = [
```

Comment:
nit: this could probably be placed in the constants folder and imported to be used here. It makes it easier to use in other places in code while only have a single source of truth to maintain